pages/components/avatar/avatar.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<script
+			src="https://kit.fontawesome.com/f9a7380946.js"
+			crossorigin="anonymous"
+		></script>
+		<link rel="stylesheet" href="../../../style.css" />
+		<link rel="stylesheet" href="../../docs.css" />
+		<link rel="stylesheet" href="./avatar.css" />
```

Comment:
Try to import only a single stylesheet in an HTML and if you require styles from other stylesheets add them in your that stylesheet that you have imported in your HTML using import.